### DISCUSSION TITLE
❓ Questions / Help / Support where can I get the latest vad model?

### DISCUSSION THREAD

**Discussion by computervisionlearner** [snakers4/silero-vad#589] [Q&A] [answered]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

I have read the whole docs of this project, but cannot find any link to download vad model. Could you please tell me how to download the vad onnx model?

**Comment by computervisionlearner** [comment]
after I read the example codes, I guess the vad model was downloaded automatically after I execute `pip install silero-vad`?

maybe the path of onnx model is `/root/anaconda3/lib/python3.10/site-packages/silero_vad/data/silero_vad.onnx`?

**Comment by snakers4** [comment, marked as answer]
If you need just the models, the are here https://github.com/snakers4/silero-vad/tree/master/src/silero_vad/data

**Comment by ManuSrivastava1** [comment]
Hello, compliments on the release of V5. I wanted to ask if the version available on torch.hub the latest version of an older one? thank you.

**Reply by snakers4** [reply]
the latest model can be pulled from repo via torch.hub or onnx or pip package
